.Glass/glass.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Question: Do we want git to ignore this file?

Reply:
Yes, we'll get git to ignore this file.